src/components/DateDistance/README.md

```diff
@@ -0,0 +1,14 @@
+Default usage:
```

Comment:
Are you able to add some margin in between the "View Code" tab and the code portion globally? I find it hard to see the tab's bottom border.

<img width="727" alt="screen shot 2018-04-16 at 11 08 31 am" src="https://user-images.githubusercontent.com/3766511/38818064-cf01548e-4167-11e8-942d-b39cff32ed0c.png">

Reply:
<img width="519" alt="screen shot 2018-04-16 at 4 15 21 pm" src="https://user-images.githubusercontent.com/285899/38835663-697fdc8c-4191-11e8-83d0-963d3b31730f.png">